src/utils/index.js

```diff
@@ -91,22 +91,15 @@ export const blockToDate = (goalBlock, height) => {
   return new Date(diff * 180000 + Date.now());
 };
 
-export const wrapTry = async (promise, store) => {
+export const wrapTry = async (promise) => {
+  // eslint-disable-next-line no-useless-catch
   try {
     return promise.then((res) => {
-      if (!res) {
-        if (store) { store.commit('setBackendStatus', false); }
-        return null;
-      }
-      if (store) { store.commit('setBackendStatus', true); }
-      if (!res.ok) throw new Error(`Request failed with ${res.status}`);
+      if (!res) throw new Error('Responce object is undefined');
```

Comment:
remove `.catch((err) => console.error(err))` in https://github.com/aeternity/superhero-ui/blob/8650d831804fe2e98e7862aa3de0109605bc78a2/src/utils/backend.js#L6 instead

Reply:
thanks, done

Reply:
```suggestion
```